lib/compat.php

```diff
@@ -228,6 +228,27 @@ function gutenberg_get_post_type_capabilities( $user, $name, $request ) {
 	return $value;
 }
 
+/**
+ * Make sure oEmbed REST Requests apply the WP Embed security mechanism for WordPress embeds.
+ *
+ * @see  https://core.trac.wordpress.org/ticket/32522
```

Comment:
Should a comment be made to this ticket (or a new ticket) describing what's being added here and why?

Reply:
Sure, I just opened this [ticket](https://core.trac.wordpress.org/ticket/43136) in WordPress Core Trac.